policyengine_us/variables/gov/states/ca/cdss/tanf/child_care/eligibility/ca_calworks_child_care_property_value_eligible.py

```diff
@@ -0,0 +1,33 @@
+from policyengine_us.model_api import *
+
+
+class ca_calworks_child_care_property_value_eligible(Variable):
+    value_type = bool
+    entity = SPMUnit
+    label = "Eligible child for the California CalWORKs Child Care based on the property value"
+    definition_period = YEAR
+    defined_for = StateCode.CA
+    reference = "https://dpss.lacounty.gov/en/cash/calworks.html"
+
+    def formula(spm_unit, period, parameters):
+        assets = spm_unit("spm_unit_cash_assets", period)
+        # The limit increases if it least one person is aged or disabled
+        person = spm_unit.members
+        age = person("age", period)
+        disabled = person("is_disabled", period)
+        p = parameters(
+            period
+        ).gov.states.ca.cdss.tanf.child_care.eligibility.resource_limit
+        aged = age >= p.property.age_threshold
+        aged_or_disabled_person = spm_unit.any(aged | disabled)
+        value_limit = where(
+            aged_or_disabled_person,
+            p.property.aged_or_disabled,
+            p.property.base,
+        )
+        # The excess of the vehicle value over the vehicle value limit is accounted towards
```

Comment:
why do we need to do this? if they exceed the vehicle value limit aren't they disqualified anyway? is it because people who reside in their vehicles are exempt from the vehicle value limit?

Reply:
I assumed so at first but now looking through the documentation I do not see anything that would suggest that this is not obsolete, homelessness does not affect the vehicle value limitations.  

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/555996ee-8008-4d8d-b763-915c8f56759d)